extensions/src/main/java/io/wcm/caconfig/extensions/contextpath/impl/AbsoluteParentContextPathStrategy.java

```diff
@@ -110,8 +117,12 @@ void activate(Config config) {
     if (config.levels() != null) {
       for (int level : config.levels()) {
         levels.add(level);
+        if (level >= unlimitedLevelStart) {
```

Comment:
Wouldn’t it make more sense to make the smallest limit the unlimitedLevelStart?

Reply:
After having a second thought although using the largest limit as lower bound gives more flexibility (i.e. allows to skip certain levels) but is more complicated. @stefanseifert Do you have a good use case where one would be interested in e.g. covering levels 2,4 and all above but not level 3?

Reply:
in my pov it makes more sense to use the highest level as starting point - using the lowest level would render all other levels that are entered superfluous.
in the first few hierarchy levels you may have one that you want to skip (e.g. pure structure hierarchy levels without business meaning).